src/components/suggestions.ts

```diff
@@ -57,6 +57,26 @@ export const addSuggestion = async (
   );
 };
 
+export const updateSuggestionCron = async (state = 'pending', oldState = 'created'): Promise<boolean> => {
```

Comment:
with the above comment, this can just be a simple `updateSuggestionState`, which could take in a list of suggestion IDs and the target state as parameters

Reply:
this function is nice because your next ticket could reuse this function